src/clarity-angular/datagrid/datagrid.spec.ts

```diff
@@ -77,29 +78,32 @@ export default function (): void {
                     expect(context.testComponent.nbRefreshed).toBe(1);
                 });
 
-                it("emits once when the sort order changes", function () {
+                it("emits once when the sort order changes",  fakeAsync(() => {
                     context.testComponent.nbRefreshed = 0;
                     let sort: Sort = context.getClarityProvider(Sort);
                     sort.toggle(new TestComparator());
+                    tick(1000);
```

Comment:
Ugh. That's why debounce is so horrible in a plain component library. Debouncing should happen on the app level.